_entries/02-01 challenge1.md

```diff
@@ -61,30 +65,12 @@ az group create --name <resource-group> --location <region>
 
   {% collapsible %}
  
-  AKS clusters that support the cluster autoscaler must use virtual machine scale sets and run Kubernetes version *1.12.4* or later. This scale set support is in preview. To opt in and create clusters that use scale sets, first install the *aks-preview* Azure CLI extension using the `az extension add` command, as shown in the following example:
+  AKS clusters that support the cluster autoscaler must use virtual machine scale sets and run Kubernetes version *1.12.4* or later. Cluster autoscaler support is in preview. To opt in and create clusters that use this feature, first install the *aks-preview* Azure CLI extension using the `az extension add` command, as shown in the following example:
 
   ```sh
   az extension add --name aks-preview
   ```
 
-  To create an AKS cluster that uses scale sets, you must also enable a feature flag on your subscription. To register the *VMSSPreview* feature flag, use the `az feature register` command as shown in the following example:
-
-  ```sh
-  az feature register --name VMSSPreview --namespace Microsoft.ContainerService
-  ```
-
-  It takes a few minutes for the status to show *Registered*. You can check on the registration status using the `az feature list` command:
-
-  ```sh
-  az feature list -o table --query "[?contains(name, 'Microsoft.ContainerService/VMSSPreview')].{Name:name,State:properties.state}"
-  ```
-
-  When ready, refresh the registration of the *Microsoft.ContainerService* resource provider using the `az provider register` command:
-
-  ```sh
-  az provider register --namespace Microsoft.ContainerService
-  ```
-
```

Comment:
Are those not needed anymore?

Reply:
Not since VMSS went GA last week. I've verified this with a fresh Azure account and subscription, no longer need to register those providers. Good thing too as they can take 30 mins to register. You still need the preview extension however for the cluster auto scaler feature